Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -976,6 +983,48 @@ def get_incident_command(client: Client, args: dict[str, Any]) -> CommandResults
     return command_results
 
 
+def get_external_websites_command(client: Client, args: Dict[str, Any]) -> CommandResults:
```

Comment:
```suggestion
def list_external_websites_command(client: Client, args: Dict[str, Any]) -> CommandResults:
```
Hope this doesn't seem like a nit, but our pattern is to use `list_` when looking up multiple items and `get_` for pulling information on a single item.  Because this is the former, the command should be changed to !asm-list-external-websites